eslint.config.js

```diff
@@ -0,0 +1,22 @@
+// @ts-check
+
+import eslint from '@eslint/js'
+import tseslint from 'typescript-eslint'
+import globals from 'globals'
+
+export default tseslint.config(
+    eslint.configs.recommended,
+    ...tseslint.configs.recommended,
+    { files: ['src/**/*.ts'] },
+    {
+        ignores: ['public/', '**/moses.js', '**/_*.ts', 'build/']
+    },
+    { files: ['static/js/**.js'], languageOptions: { globals: globals.browser } },
+    {
+        rules: {
+            '@typescript-eslint/no-explicit-any': 'warn',
+            '@typescript-eslint/no-unused-vars': 'warn',
```

Comment:
To let us make eslint happy
```suggestion
            '@typescript-eslint/no-unused-vars': [
                'warn',
                {
                    argsIgnorePattern: '^_',
                    varsIgnorePattern: '^_',
                    caughtErrorsIgnorePattern: '^_',
                },
            ],
```